siad/daemon.go

```diff
@@ -125,6 +139,7 @@ func startDaemon(config Config) (err error) {
 		if err != nil {
 			return err
 		}
+		defer g.Close()
```

Comment:
`srv` does not do module initialization.  Closing should be handled by the initializer, in this case `siad`.